tools/initdata.yml

```
@@ -24,7 +24,9 @@ users:
 
   - id: "2753fcf9-ab6a-426e-b1d1-e4e16470a5d0"
     params:
-      email: catalogue.demo2@yopmail.com # This user is linked to two organizations
+      organization_siret: "13001653800014"
+      # In MonComptePro, a user linked to two organizations was created with this email.
```

Ou bien ceci... ? "auth.api.gouv.fr" c'est le nom de domaine (inchangé) du branding "Compte DataPass" devenu "MonComptePro"

```suggestion
      # In auth-staging, a user linked to two organizations was created with this email.
```